new-chatbot-app/src/library-api/libcal-authorization/libcal-authorization.service.spec.ts

```diff
@@ -0,0 +1,82 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { LibcalAuthorizationService } from './libcal-authorization.service';
+import { HttpModule, HttpService } from '@nestjs/axios';
+import { of } from 'rxjs';
+import { AxiosResponse } from 'axios';
+import { SharedModule } from '../../shared/shared.module';
+import { LibraryApiModule } from '../library-api.module';
+
+describe('LibcalAuthorizationService', () => {
+  let service: LibcalAuthorizationService;
+  let httpService: HttpService;
+
+  beforeEach(async () => {
+    const module: TestingModule = await Test.createTestingModule({
+      imports: [HttpModule, LibraryApiModule, SharedModule],
+      providers: [LibcalAuthorizationService],
+    }).compile();
+
+    service = module.get<LibcalAuthorizationService>(
+      LibcalAuthorizationService
+    );
```

Comment:
httpService = module.get<HttpService>(HttpService);

Reply:
Didn't I put in from line 21-23?